app/views/playlist.js

```diff
@@ -3,8 +3,18 @@ import application from '../application'
 
 const PlaylistView = Mn.ItemView.extend({
 
+  template: require('./templates/popupPlaylists'),
+
   tagName: 'li',
 
+  ui: {
+    'play': '.play'
```

Comment:
This will go, as for the next comment, the double-click should occur on the `li`.